frontend/src/components/search/SearchResultsView.tsx

```diff
@@ -0,0 +1,137 @@
+import withFeatureFlag from "src/services/featureFlags/withFeatureFlag";
```

Comment:
this file is here to facilitate switching the experience from the legacy view to the table view. It can be removed when we deprecate the legacy view, so I'm not bothering to test it